asm-multiproject-patch/resources/istio-operator.yaml

```diff
@@ -0,0 +1,51 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+metadata:
+  annotations:
+    gke.io/cluster: "gke://PROJECT_ID/us-central1-c/asm-cluster" # {"$ref":"#/definitions/io.k8s.cli.substitutions.gkeio-cluster"}
+  name: default-istio-operator
+spec:
+  profile: asm-gcp-multiproject # {"$ref":"#/definitions/io.k8s.cli.setters.anthos.servicemesh.profile"}
+  hub: gcr.io/gke-release/asm # {"$ref":"#/definitions/io.k8s.cli.setters.anthos.servicemesh.hub"}
+  tag: 1.6.5-asm.7 # {"$ref":"#/definitions/io.k8s.cli.setters.anthos.servicemesh.tag"}
+  components:
+    pilot:
+      k8s:
+        env:
+        - name: PILOT_SKIP_VALIDATE_TRUST_DOMAIN
```

Comment:
Update: This one is no longer needed after we have the cherry-picked a fix into 1.6-asm branch. @yangminzhu right?

Reply:
Let me quickly verify this before the end of today without setting this plus including the fix, see if that works.

Reply:
Yes. You can now ignore the `PILOT_SKIP_VALIDATE_TRUST_DOMAIN`.  It should not matter after another fix is merged.

https://gke-internal-review.googlesource.com/c/istio/istio/+/160952

Reply:
Awesome. Thanks a lot!